services/fxa/serverless.yml

```diff
@@ -186,113 +93,207 @@ custom:
           - execute-api:/*/*/plans
         Condition:
           IpAddress:
-            aws:SourceIp: ${file(./whitelist.yml):payments.${self:provider.stage}}
+            aws:SourceIp: ${file(whitelist.yml):payments.${self:provider.stage}}
       - Effect: Allow
         Principal: "*"
         Action: execute-api:Invoke
         Resource:
           - execute-api:/*/*/hub
         Condition:
           IpAddress:
-            aws:SourceIp: ${file(./whitelist.yml):hub}
+            aws:SourceIp: ${file(whitelist.yml):hub}
+
+plugins:
+  - serverless-python-requirements
+  - serverless-domain-manager
+  - serverless-plugin-tracing
+  - serverless-dynamodb-local
+  - serverless-package-external
+  - serverless-offline
+provider:
+  name: aws
+  runtime: python3.7
+  region: us-west-2
+  stage: ${opt:stage}
+  stackName: ${self:custom.prefix}-stack
+  apiName: ${self:custom.prefix}-apigw
+  deploymentPrefix: ${self:custom.prefix}
+  endpointType: regional
+  logRetentionInDays: 90
+  # Logging
+  logs:
+    # NOTE: https://github.com/serverless/serverless/issues/6112
+    restApi: true
+  # Tracing
+  tracing:
+    lambda: true
+    apiGateway: true
```

Comment:
Add tracing around both Lambda and API Gateways.

Reply:
where does that go to? a log stream? which?